composer.json

```diff
@@ -9,13 +9,12 @@
         }
     ],
     "require": {
-        "php": "^5.3.3|^7.0",
+        "php": "^7.3 || ^8.0.0",
         "webmozart/path-util": "^2.2"
     },
     "require-dev": {
-        "symfony/filesystem": "^2.5",
-        "phpunit/phpunit": "^4.6",
-        "sebastian/version": "^1.0.1"
+        "symfony/filesystem": "^5.1",
+        "phpunit/phpunit": "^8.0"
```

Comment:
Why not `9.x`? Should work with 7.3 - 8.0

Reply:
For now, this patch only bumps to `8.x`.

I'll do further dependency upgrades (for all dependencies) later